pages/_app.tsx

```diff
@@ -29,7 +61,38 @@ function MyApp({ Component, pageProps }: AppProps) {
               />
             </Head>
             <Navbar />
-            <Component {...pageProps} />
+            <Notification visible={!cookiesAccepted} severity="info">
+              <div className="flex flex-wrap sm:gap-20 gap-5">
+                <p>
+                  We use cookies to ensure you get the best experience on our
```

Comment:
what cookies do we use?

Reply:
No cookies just gather data I guess, What do you propose ?

Reply:
just asking them to accept the privacy policy should be enough then :)